### PR TITLE
Blackjack game using C++ (workshop bounty)

### PR DESCRIPTION
**Pull request by chrisrama**
I am pretty much done, I now saw those capitalizations, the grammar I had. I am editing my video and should be submitted soon. 
If there is any feedback, I am here :)

### CODE REVIEW

**Comment by chrisrama** [comment]
@sampoder how is this so far

File: workshops/backjack/README.md [hunk 1]
```diff
@@ -0,0 +1,1528 @@
...
++ Inside the project, I have included the c time library (` <ctime> ` ), this header file will be called when we need to generate random numbers which will be the card numbers for the players. For more about how to use the this` library, you can found it [Here](http://www.cplusplus.com/reference/ctime/)
++ I also called the vector library (` <vector> `), because we will be using vectors instead of arrays to keep all the player and the dealer’s cards.
++ We are not going to be using ` std:: ` to declare a string variable or output and input messages, instead we will be using ` using namespace std; ` to inform the program that we won't be using ` std:: ` rather a namespace.
+
+ 
+ ```c++
+ /* Declaration section: this will be used to declare variables and functions */ 
+
+  void start_The_Game();                   /* declare start_The_Game function */
+    int Pl_card1, Pl_card2, Pl_card_cont;      /* used to stored the card number for the player*/
```
Comment (sampoder) [inline workshops/backjack/README.md:53]: why is this all indented in one, and some in two, should all have 0 tabs

File: workshops/backjack/README.md [hunk 2]
```diff
@@ -0,0 +1,1528 @@
...
+string player1;          		      /* this how it should look */
+
+ ```
+Inside the main function, we have asked the player for their name, let's then take their name using the ` getline() ` function which will take the whole line as the input, as we want the player's full name.
+
+Inside  getline function we want to get the input and stored it inside ` player1 `, so we will use  ` cin `to get input and ` player1 ` together as ` getline(cin,player1) ` : 
+
+ ```c++     
+ 
+   getline(cin,player1);                                             /* this should be written under the last line inside the main function */
```
Comment (sampoder) [inline workshops/backjack/README.md:122]: Should be at 0 tabs, should have no new lines around it

File: workshops/backjack/README.md [hunk 3]
```diff
@@ -0,0 +1,1528 @@
...
+
+```C++
+/* Definition section: this section will be used to define all declared function*/
+
+void start_The_Game() {  
+   cout << "\n           Blackjack Game      \n";					 /* again print out the title to the game */
+   cout << "\n++++++++++++++++++++++++++++++++\n";					 /* print out a line using plus sign */
+   cout << "\n Welcome to the game.\n";							 /* print a welcome message */
+   cout << "\n Player of this round is: " << player1 << "\n\n"; 			 /* print the name of player */
+  }
```
Comment (sampoder) [inline workshops/backjack/README.md:150]: This `}` should be at an indent level 0

**Review by sampoder — CHANGES_REQUESTED** [review]
The code indentation and formatting is still lacking and making this hard to read. 

I've pointed out some examples:

**Comment by chrisrama** [comment]
let me try and see, what I can do but that how mostly, some c++ code are written

File: workshops/backjack/README.md (hunk 3 above)
Comment (chrisrama) [reply workshops/backjack/README.md:150]: @sampoder when it come to if statement I will have to use tab right?, so it readeable

**Pull request closed without merging**